src/lib/components/Alert/Alert.tsx

```diff
@@ -11,6 +11,7 @@ export interface AlertProps extends PropsWithChildren<Omit<ComponentProps<'div'>
   onDismiss?: boolean | (() => void);
   rounded?: boolean;
   withBorderAccent?: boolean;
+  className?: string;
```

Comment:
Since it extends `ComponentProps<'div'>`, you don't need to specify the `className` property.